src/components/PropertyMap.tsx

```diff
@@ -235,14 +243,19 @@ const PropertyMap: FC<PropertyMapProps> = ({
 
   useEffect(() => {
     if (map) {
-      // Add Legend Control
-      if (!legendRef.current) {
-        legendRef.current = new LegendControl({
-          layers: ["vacant_properties_tiles_polygons"],
-        });
-
-        // Note: this is a hack to get around the typescript error because we are using incompatible packages. We should write our own legend
-        map.addControl(legendRef.current as unknown as IControl, "bottom-left");
+      // Add info icon to legend on map load
+      const legendSummary = document.getElementById("legend-summary");
+      if (legendSummary) {
+        summaryInfo = createPortal(
+          <Tooltip content="We prioritize properties based on how much they can reduce gun violence considering the vacancy, gun violence, cleanliness, and tree canopy.">
+            <Info
+              alt="Priority Info"
+              className="text-gray-500 cursor-pointer"
+              tabIndex={0}
+            />
+          </Tooltip>,
```

Comment:
<img width="1043" alt="Screenshot 2024-03-24 at 6 11 11 PM" src="https://github.com/CodeForPhilly/vacant-lots-proj/assets/43150970/0c43a3b7-b954-4065-896f-a11c727fef17">
This is what it looks like now, should I make the tooltip a smaller width?